Description: Fix documentation URL
 Use web.archive.org until project website will be repaired.
Author: Boris Pek <tehnick-8@yandex.ru>
Last-Update: 2016-01-24
Forwarded: not-needed
Bug: https://github.com/brezerk/q4wine/issues/45
Bug-Debian: http://bugs.debian.org/761968
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/q4wine/+bug/1332725


--- a/config.h.cmake
+++ b/config.h.cmake
@@ -45,7 +45,7 @@
 
 #define APP_NAME "Q4Wine"
 #define APP_SHORT_NAME "q4wine"
-#define APP_WEBSITE "q4wine.brezblock.org.ua"
+#define APP_WEBSITE "web.archive.org/web/20131204020055/http://q4wine.brezblock.org.ua"
 #define APP_BUG_EMAIL "brezerk@gmail.com"
 
 #define APPDB_EXPORT_VERSION "1.0"
@@ -57,7 +57,7 @@
 #define RUN_STRING_TEMPLATE_PRIMUSRUN "%CONSOLE_BIN% %CONSOLE_ARGS% %ENV_BIN% %ENV_ARGS% /bin/sh -c \"%WORK_DIR% %SET_NICE% primusrun %WINE_BIN% %VIRTUAL_DESKTOP% %PROGRAM_BIN% %PROGRAM_ARGS% 2>&1 \""
 
 
-#define APPDB_HOSTNAME "q4wine.brezblock.org.ua"
+#define APPDB_HOSTNAME "web.archive.org/web/20131204020055/http://q4wine.brezblock.org.ua"
 #define BUGS_HOSTNAME "bugs.winehq.org"
 
 #define APPDB_PORT 80
diff --git a/src/q4wine-lib/q4wine-lib.cpp b/src/q4wine-lib/q4wine-lib.cpp
index 0fe09c1..962c4aa 100644
--- a/src/q4wine-lib/q4wine-lib.cpp
+++ b/src/q4wine-lib/q4wine-lib.cpp
@@ -1608,7 +1608,7 @@
             QString url="http://";
             url.append(APP_WEBSITE);
             url.append("/documentation/");
-            url.append(this->getLang());
+            url.append("en_us");
             url.append("/");
             url.append(rawurl);
 
